css-typed-om/Overview.bs

```diff
@@ -2847,10 +2847,49 @@ of serializations of the contained {{CSSTransformComponent}} objects.
 
 {{CSSFontFaceValue}} Serialization {#fontfacevalue-serialization}
 -----------------------------------------------------------------
-
+ 
 {{CSSFontFaceValue}} objects are serialized to the value of their contained
 {{CSSFontFaceValue/fontFamilyName}}.
 
+Units and Values Serialization {#unitsandvalues-serialization}
+--------------------------------------------------------------
+
+This section covers how to serialize the various unit types in CSSOM
+
+<div algorithm="serialize a 'length' value">
+
+To <dfn>serialize a 'length' value</dfn>, run these steps:
+    1. Let |values| initially be the empty [=list=].
+    2. If the <length> value has a '-' sign character append it to |values|.
```

Comment:
Lengths don't have characters; look for it being negative instead, and then set it to its absolute value.